assets/js/admin.min.js

```diff
@@ -696,4 +696,19 @@ XH.on(window, "load", function () {
     if (newPasswordInput) {
         XH.addInputEventListener(newPasswordInput, showScore);
     }
+
+    /*
+    * Replaces the PNG tooltip image with an SVG image
+    * @since 1.7.3
+    */
+    var replTooltipIcon = document.getElementsByClassName("pl_tooltip");
+    for (var i = 0; i < replTooltipIcon.length; i++) {
+        var image = replTooltipIcon[i].getElementsByTagName('img')[0];
+        var imgSrc = image.src;
+        if (imgSrc.indexOf('assets') != -1) {
+            console.log(imgSrc.indexOf('assets'));
```

Comment:
Ich denke, dieser debug output sollte entfernt werden. :)

Reply:
Ah, du meinst Zeile 709 die Konsole-Ausgabe?
Ja, das ist ein Überbleibsel.
@olape-git

Reply:
also 708 und 709 raus?

Reply:
Ich denke nur 709 (console.log) löschen.
Die Bedingung in 708 muss wohl drin bleiben.

Reply:
Ja, mir ging es nur um `console.log`. Die Ausgabe ist nicht schlimm, aber unnötigt, und bei alten IEs ist damals das ganze JS abgeschmiert.

Reply:
Ja, wenn man sich die ganze Datei mal ansieht kommt man auch alleine drauf. Also nur 709.